test/Macros/macro_self.swift

```diff
@@ -0,0 +1,27 @@
+// RUN: %target-swift-frontend -parse %s -verify
+
+@freestanding(expression) // expected-error {{expected expression}}
+macro self() = #externalMacro(module: "MacroDefinition", type: "InvalidMacro")
```

Comment:
This error message doesn't look right.

Reply:
I thought the same but I don’t think it’s worth improving it. This is silly code that I don’t expect anyone to write and if we think it’s worth improving the diagnostics for this, we should do so in the new parser.